x-pack/plugins/security_solution/public/common/components/toolbar/bulk_actions/alert_bulk_tags.tsx

```diff
@@ -24,6 +25,7 @@ interface BulkAlertTagsPanelComponentProps {
   refresh?: () => void;
   clearSelection?: () => void;
   closePopoverMenu: () => void;
+  onSubmit: SetAlertTagsFunc;
 }
 const BulkAlertTagsPanelComponent: React.FC<BulkAlertTagsPanelComponentProps> = ({
```

Comment:
It seems you still have a bad setState call somewhere in this component. To reproduce, open the dev console, add a tag to an alert.  You will see this error:

```
react_devtools_backend_compact.js:2367 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    at BulkAlertTagsPanelComponent (http://localhost:5601/kbn/9007199254740991/bundles/plugin/securitySolution/1.0.0/securitySolution.chunk.lazy_register_alerts_table_configuration.js:107969:3)
    at div
    at EuiResizeObserver (http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:122904:81)
    at div
    at EuiContextMenuPanel (http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:78775:81)
    at div
    at EuiContextMenu (http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:78253:81)
    at div
    at EuiMutationObserver (http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:122667:81)
    at div
    at http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:160823:73
    at EuiPanel (http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:126691:23)
    at http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:160823:73
    at EuiPopoverPanel (http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:127994:23)
    at div
    at http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:361576:59
    at FocusLockUI (http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:357879:71)
    at http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:358782:60
    at EuiFocusTrap (http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:100087:81)
    at EuiPortal (http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:128462:81)
    at div
    at http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:160823:73
    at EuiPopover (http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:127230:81)
    at div
    at forwardRef (http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:418712:12)
    at div
    at AlertContextMenuComponent (http://localhost:5601/kbn/9007199254740991/bundles/plugin/securitySolution/1.0.0/securitySolution.chunk.lazy_register_alerts_table_configuration.js:143217:3)
    at ConnectFunction (http://localhost:5601/kbn/9007199254740991/bundles/plugin/securitySolution/1.0.0/securitySolution.chunk.3.js:422:41)
    at div
    at forwardRef (http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:418712:12)
    at ActionsComponent (http://localhost:5601/kbn/9007199254740991/bundles/plugin/securitySolution/1.0.0/securitySolution.chunk.lazy_register_alerts_table_configuration.js:95747:3)
    at RowActionComponent (http://localhost:5601/kbn/9007199254740991/bundles/plugin/securitySolution/1.0.0/securitySolution.chunk.lazy_register_alerts_table_configuration.js:85797:3)
    at div
    at http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:160823:73
    at http://localhost:5601/kbn/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:98843:23
    at rowCellRender (http://localhost:5601/kbn/9007199254740991/bundles/plugin/triggersActionsUi/1.0.0/triggersActionsUi.chunk.25.js:871:13)
```

Could you please double check the async behavior in this component and ensure that any state updates are done while the component is still mounted.

Reply:
Good news, I thought I had your branch checked out but alas, I did not. Checked on your branch and this error is cleared! Hooray! Going to resolve this convo, sorry if it held you up at all